package.json

```diff
@@ -2,61 +2,73 @@
   "name": "bind-event-listener",
   "version": "3.0.0",
   "private": false,
-  "description": "Making binding and unbinding DOM events easier",
+  "type": "module",
+  "description": "Safe DOM Event binding and unbinding",
   "author": "Alex Reardon <alexreardon@gmail.com>",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/alexreardon/bind-event-listener.git"
   },
   "keywords": [
+    "event listener",
+    "event handler",
+    "dom events",
     "dom",
     "ui",
-    "event listener",
-    "event handler"
+    "bind"
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
   "sideEffects": false,
+  "main": "./dist/esm/index.js",
+  "module": "./dist/esm/index.js",
```

Comment:
this serves no purpose here, everything is already in ESM so we don't need this extra file. It was originally introduced when `main` was pointing to CJS files and people wanted to have some ESM entry